reactos/drivers/usb/usbport/usbport.h

```diff
@@ -565,19 +565,20 @@ USBPORT_FindCompanionControllers(
   IN BOOLEAN IsObRefer,
   IN BOOLEAN IsFDOsReturned);
 
+VOID
+NTAPI
+USBPORT_InvalidateControllerHandler(
+  IN PDEVICE_OBJECT FdoDevice,
+  IN ULONG Type);
+
 /* debug.c */
 ULONG
 NTAPI
```

Comment:
The compilers did not say anything.
It's just a stub for compatibility and I do not know if there will ever be anything in it someday.